lerna.json

```diff
@@ -1,7 +1,5 @@
 {
-	"lerna": "2.0.0",
-	"packages": [
-		"packages/*"
-	],
+	"lerna": "2.8.0",
```

Comment:
This remains at `2.0.0`, this is a Lerna specific version, not the npm module packaged version.

Reply:
That should be updated to update lerna. Otherwise, there will be a version mismatch.